mathics/builtin/base.py

```diff
@@ -1012,28 +1012,28 @@ def from_expression(expr):
         """
 
         if isinstance(expr, Integer):
-            py_n = expr.get_int_value()
+            py_n = expr.value
             if py_n >= 0:
                 return CountableInteger(py_n, upper_limit=False)
             else:
                 raise NegativeIntegerException()
         elif expr.get_head_name() == "System`UpTo":
-            if len(expr.leaves) != 1:
-                raise MessageException("UpTo", "argx", len(expr.leaves))
+            if len(expr.elements) != 1:
+                raise MessageException("UpTo", "argx", len(expr.elements))
             else:
-                n = expr.leaves[0]
+                n = expr.elements[0]
                 if isinstance(n, Integer):
-                    py_n = n.get_int_value()
+                    py_n = n.value
                     if py_n < 0:
                         raise MessageException("UpTo", "innf", expr)
                     else:
                         return CountableInteger(py_n, upper_limit=True)
                 elif CountableInteger._support_infinity:
                     if (
                         n.get_head_name() == "System`DirectedInfinity"
-                        and len(n.leaves) == 1
+                        and len(n.elements) == 1
                     ):
-                        if n.leaves[0].get_int_value() > 0:
+                        if n.elements[0].get_int_value() > 0:
```

Comment:
`.value` here.

Reply:
The broader context: 

```
                if isinstance(n, Integer):
                    ....
                elif CountableInteger._support_infinity:
                    if (...):
                        if n.elements[0].get_int_value() > 0:
```

So no, not here just yet.  I still think `get_int_value()` in general is a name from the bad-old-days, but addressing this is a separate concern and set of PRs.